src/System.Private.ServiceModel/tests/Scenarios/Binding/Custom/CustomBindingTests.cs

```diff
@@ -4,13 +4,14 @@
 using System;
 using System.ServiceModel;
 using System.ServiceModel.Channels;
+using System.ServiceModel.Tests.Common;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using TestTypes;
 using Xunit;
 
-public static class CustomBindingTests
+public class CustomBindingTests : IClassFixture<BridgeTestFixture>
 {
```

Comment:
I am fairly sure the xunit wrapper we use in ToF does not support IClassFixture. I would like to hear the reasons why all the tests need to modified to initialize like this, as opposed to the static BridgeClient approach.